instruqt-tracks/terraform-intro-gcp/track_scripts/setup-workstation

```diff
@@ -43,9 +43,11 @@ cp ${GITDIR}/exercises/outputs.tf.start ${GITDIR}/outputs.tf
 mv ${GITDIR}/terraform.tfvars.example ${GITDIR}/terraform.tfvars
 
 # Store our project ID as a Terraform env var
-export TF_VAR_project=$INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID
-grep $INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID /root/.bashrc || echo "export TF_VAR_project=\"$INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID\"" >> /root/.bashrc
-export GOOGLE_CREDENTIALS=$(echo $INSTRUQT_GCP_PROJECT_GCP_PROJECT_SERVICE_ACCOUNT_KEY | base64 -d | jq 'tostring')
-echo "export GOOGLE_CREDENTIALS=$GOOGLE_CREDENTIALS" >> /root/.bashrc
+echo "TF_VAR_project=\"$INSTRUQT_GCP_PROJECT_GCP_PROJECT_PROJECT_ID\"" >> /root/.bashrc
+echo "export TF_VAR_project" >> /root/.bashrc
+GOOGLE_CREDENTIALS=$(echo "$INSTRUQT_GCP_PROJECT_GCP_PROJECT_SERVICE_ACCOUNT_KEY" | base64 -d | jq 'tostring')
+# Do not quote '$GOOGLE_CREDENTIALS' because `jq 'tostring'` does that for us
+echo "GOOGLE_CREDENTIALS=$GOOGLE_CREDENTIALS" >> /root/.bashrc
+echo "export GOOGLE_CREDENTIALS" >> /root/.bashrc
```

Comment:
LInes 46-51 we follow `shellcheck`s advice to set the variables on one line, then export them on the next. `export` can mask errors in any sub-command used to set the shell's variable because `export` always succeeds.